frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -49,6 +56,17 @@ const plugin: Plugin<ConsumedExtensions> = [
       title: 'Persistent Storage',
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: `/k8s/ns/:ns/${ClusterServiceVersionModel.plural}/:appName/${apiObjectRef}/~new`,
```

Comment:
It would be better to provide a first-class extension point for creation forms in a future follow-on.